Makefile

```diff
@@ -32,9 +33,9 @@ default: build
 .PHONY : build-static test install-lint-deps lint go-test fmt fmtcheck verify-vendor verify create-release-files release
 build-static: ## Build garm statically
 	@echo Building garm
-	docker build --tag $(IMAGE_TAG) -f Dockerfile.build-static .
+	$(IMAGE_BUILDER) build -v /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt --tag $(IMAGE_TAG) -f Dockerfile.build-static .
```

Comment:
This will most likely break docker. See:

https://github.com/cloudbase/garm/blob/bb45324dcd929193ec31a06bcbc450a114668fb4/Makefile#L25-L27

What would probably work on both podman and docker (provided podman implements the `--mount` option for thr `RUN` stanza is changing the Dockerfile with something like this:

```Dockerfile
RUN --mount=type=bind,source=/etc/ssl/certs/ca-certificates.crt,target=/etc/ssl/certs/ca-certificates.crt REST_OF_COMMANDS_GO_HERE
```

Reply:
Damn, this change was not expected to be included in this P/R. I will fix that.